src/components/request/RequestView.tsx

```
@@ -7,7 +11,29 @@ const RequestView = ({
   routineDetail,
   imagePath,
 }: RoutineRequestDetail) => {
-  const closeModal = useModalStore((state) => state.close);
+  const user = useAuthStore((state) => state.user);
+  const [comment, setComment] = useState('');
```

ESLint

[error] 'comment' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)